model-optimizer/extensions/front/tf/KerasRNNReplacer.py

```diff
@@ -0,0 +1,237 @@
+    tensor_list_get_item_node.out_port(0).get_connection().set_source(squeeze_list_element.out_port(0))
+    rename_nodes([(tensor_list_get_item_node, tensor_list_get_item_node_name + '/AbandonedName'),
+                  (squeeze_list_element, tensor_list_get_item_node_name)])
+    unstack_placeholder_layer_id = unstack_placeholder.internal_layer_id
+    Loop.update_port_map_value_ext(loop_node.input_port_map, 'internal_layer_id', unstack_placeholder_layer_id,
+                                   'axis', 0)
+
+    # replace TensorListSetItem with Unsqueeze and use axis attribute for corresponding Result node
+    # to concatenate results from different iterations
+    unsqueeze_list_element = create_op_with_const_inputs(body_graph, Unsqueeze, {1: int64_array(0)},
+                                                         {'name': 'TensorListSetItemUnsqueeze'})
+    tensor_list_set_item_node.in_port(2).get_connection().set_destination(unsqueeze_list_element.in_port(0))
+    tensor_list_set_item_node.out_port(0).get_connection().set_source(unsqueeze_list_element.out_port(0))
+    rename_nodes([(tensor_list_set_item_node, tensor_list_set_item_node_name + '/AbandonedName'),
+                  (unsqueeze_list_element, tensor_list_set_item_node_name)])
+    list_result_node_layer_id = list_result_node.internal_layer_id
+    Loop.update_port_map_value_ext(loop_node.output_port_map, 'internal_layer_id', list_result_node_layer_id,
+                                   'axis', 0)
+
+    # 2. process locality of Loop node in the main graph to avoid unsupported operations:
+    # TensorListFromTensor, TensorListReserve, and TensorListStack
+    # remove TensorListFromTensor and pass a tensor to Loop as is
+    unstack_node.out_port(0).get_connection().set_source(unstack_node.in_port(0).get_connection().get_source())
+
+    # remove TensorListStack to by-pass the node since the result from the Loop node is already concatenated
+    stack_node.out_port(0).get_connection().set_source(stack_node.in_port(0).get_connection().get_source())
+
+    # remove TensorListReserve generating Tensor container that is no longer needed
+    graph.remove_node(list_reserve_node.id)
+
+    # connect a number of iterations with maximum_iterations
+    # create a constant network with True value for execution_condition so that IE can ignore execution condition
+    # and perform trip_counts iterations. This allows to avoid dynamism
+    loop_node.in_port(1).disconnect()
+    loop_node.in_port(strided_slice_port).get_connection().add_destination(loop_node.in_port(1))
+    for record in loop_node.output_port_map:
+        if 'purpose' in record and record['purpose'] == 'execution_condition':
+            exec_cond_layer_id = record['internal_layer_id']
+            exec_cond_node = Loop.get_body_node_by_internal_id(loop_node, exec_cond_layer_id)
+            const_true = Const(body_graph, {'value': True}).create_node()
```

Comment:
`np.array(True, dtype=np.bool)` instead of `True`. We keep all values as numpy arrays for consistency.

Reply:
Fixed.